shared/scss/popup.scss

```diff
@@ -188,16 +188,6 @@ body {
             margin-left: 12px;
         }
 
-        .site-info__protection {
-            line-height: 1.4em;
-            height: 1.2em;
-            display: block;
-            max-width: $popup__width - 125;
-            font-weight: bold;
-            overflow: hidden;
-            text-overflow: ellipsis;
-        }
```

Comment:
The element for these still exists. It was a bit off-centre, so I tried removing the top couple of rules and that fixed it. Then I tried removing all the other rules and it looked exactly the same.

I figured we shouldn't have CSS rules that do nothing, and killed the whole thing off.